jscomp/common/js_config.cppo.ml

```diff
@@ -60,7 +60,11 @@ let stdlib_path =
     Lazy.force install_dir
 #endif
 in
-  root // Literals.lib // Literals.package_name)
+  root // Literals.lib // Literals.package_name
+#ifdef BS_RELEASE_BUILD
+  // Literals.package_name
```

Comment:
this also prepares melange to install its stdlib under `$lib/PKG-NAME/melange`, just like the dune work.